src/runtime/config/configuration-qemu-se.toml.in

```diff
@@ -432,6 +432,14 @@ disable_selinux=@DEFDISABLESELINUX@
 # (default: true)
 disable_guest_selinux=@DEFDISABLEGUESTSELINUX@
 
+# Default number of queues for the tap device.
+#  <= 0 will be set to the same value as default_vcpus
+#  > 0 will be set to the value
+#  > defaultMaxVCPUs will be set to default_vcpus
+#  Leaving it at default should yield the best performance in a multi-stream scenario.
+#  If your application is single-stream it's better to set it at 1.
+default_tap_queues = 0
```

Comment:
Why not use the `network_queues` for net device? In runtime-rs, there is already a dedicated item `network_queues` used to set device queues in multi queues case. 

And I believe it is necessary to keep the configuration of runtime-rs/runtime-go versions as consistent as possible.

Reply:
Noted. Will update the PR.